test/sql/input.sql

```diff
@@ -1,5 +1,9 @@
 SELECT '[1,2,3]'::vector;
 SELECT '[-1,2,3]'::vector;
+SELECT '[1,2,3  ]'::vector;
+SELECT '[1,     2,    3  ]'::vector;
+SELECT '[1.   , 2, 3 ]'::vector(3);
+SELECT ' [1,2,3] '::vector(3);
```

Comment:
I think we can combine this into a single test

Reply:
copy that